### ISSUE TITLE
`select *` panic in subquery

### ISSUE THREAD

**Issue by skyzh** [risinglightdb/risinglight#820]
```
> create table t1(v1 int, v2 int);
> explain select * from (select v1 + v1 as v3 from t1) as left;
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/planner/rules/plan.rs:266:51
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by caicancai** [comment]
It's possible to return a default value when capturing None, I'm not sure that's a good idea

**Comment by caicancai** [comment]
> This bug has been fixed in #796. However, it has not been merged for a long time. 🥲 I can't recall too much details for now. But in my mind, this involves a tricky way how we handle the output of subquery. I might write an article to explain it in detail. (

#796 looks very cool, looking forward to the author’s article

**Comment by wangrunji0408** [comment]
fixed by #796
